app/routes/external-events.js

```diff
@@ -1,10 +1,12 @@
 const { getExternalEvents } = require('../repos/external-events')
 const { externalEventsQuerySchema, externalEventsResponseSchema } = require('../schemas/external-events')
+const { scopes } = require('../constants/auth')
 
 module.exports = {
   method: 'GET',
   path: '/external-events',
   options: {
+    auth: { scope: [scopes.admin] },
```

Comment:
Did you miss the auth scope for /events, or did it not need anything?

Reply:
It doesn't need auth scope as will allow all valid scopes